tasks/patch_wordpress.js

```diff
@@ -18,11 +18,16 @@ var request = require( 'request' )
 	, trac = require( '../lib/trac.js' )
 	, patch = require( '../lib/patch.js' )
 	, regex = require( '../lib/regex.js' )
-
+	, xmlrpc = require('xmlrpc')
+	, readline = require('readline')
 
 _.str = _.str = require('underscore.string')
 _.mixin( _.str.exports() )
 
+var rl = readline.createInterface({
```

Comment:
Elsewhere in the task, [inquirer](https://github.com/sboudrias/Inquirer.js) is used for asking questions.  What benefit do you think directly using readline has over it?